src/agent/scm/lib/scmprovider.ts

```diff
@@ -69,10 +69,28 @@ export class ScmProvider implements cm.IScmProvider {
 		
 		return paramValue;
 	}
+
+	private repoRootInFilePath(filePathInput : string) : boolean {
+		filePathInput = filePathInput.toLowerCase();
+		if(filePathInput.indexOf(cm.vars.buildSourcesDirectory.toLowerCase()) > -1 ||
```

Comment:
I think it might be more intuitive to skip the prepend only in the case where the path starts with the variable, as opposed to "contains".